api/src/api/services/brla/types.ts

```diff
@@ -210,6 +210,36 @@ export const enum SmartContractOperationType {
   BURN = 'BURN',
 }
 
+export enum KYCDocType {
+  RG = 'RG',
+  CNH = 'CNH',
+}
+
+export function isValidKYCDocType(value: string ): value is KYCDocType {
+  return Object.values(KYCDocType).includes(value as unknown as KYCDocType);
+}
+
+export interface KycLevel2Payload {
+  documentType: KYCDocType;
+}
+
+export interface KycLevel2Response {
+  id: string;
+  selfieUploadUrl: string;
+  RGFrontUploadUrl: string;
+  RGBackUploadUrl: string;
+  CNHUploadUrl: string;
```

Comment:
The capitalization is a little unfortunate. How about 
```suggestion
  rgFrontUploadUrl: string;
  rgBackUploadUrl: string;
  cnhUploadUrl: string;
```

Reply:
I had to rollback on this change, the properties are returned with this capitalization from BRLA, it may be better to leave it as-is.